modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -90,9 +90,7 @@ const setup = (editor: Editor): RenderInfo => {
   const touchPlatformClass = 'tox-platform-touch';
   const deviceClasses = isTouch ? [ touchPlatformClass ] : [];
   const isToolbarBottom = Settings.isToolbarLocationBottom(editor);
-  const container = Settings.getUiContainer(editor);
-
-  const toolbarIsInRoot = SugarNode.isTag('body')(container) || SugarShadowDom.isShadowRoot(container);
+  const toolbarIsInRoot = SugarNode.isTag('body')(Settings.getUiContainer(editor));
```

Comment:
As I mentioned in the call, this would probably be better if it was this:
```suggestion
  const toolbarIsInRoot = Compare.eq(SugarBody.body(), Settings.getUiContainer(editor));
```

We should also rename the variable to better match, as it's really just `isUiContainerInBody` or something like that.

- [ ] Rename variable and improve check